datajoint/__init__.py

```diff
@@ -81,7 +81,7 @@ def create_virtual_module(modulename, dbname):
     :return: the python module
     """
     mod = ModuleType(modulename)
-    s = schema(dbname, mod.__dict__)
```

Comment:
while at this, can we make a few changes?
1. rename `create_tables` to `create_schema` in the `Schema` class - the name `create_tables` actually doesn't make sense as this really controls whether the schema is going to be created in the database or not
2. get rid of the `mod.__dict__` - that will make this `schema` instance behave the same way as any one defined like `schema = dj.schema('schema_name')`.

Reply:
1. `create_tables` is already widely used and renaming it would cause problems.  Usually, the user means to prevent the creation of new tables when the database schema already exists.  I think it makes senes to keep things as they are now.
2. I do not believe that omitting `mod.__dict__` will do what you describe.  We are creating a module here, not a schema.

Reply:
After an off-line discussion, we agreed that `create_if_missing` will replace `create_tables` but in a future release.